packages/flutter_tool_api/test/extension_test.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Note that we'll need tests that the core tool fails gracefully when the extension isolates misbehave in various ways.